libs/base/ui/form/checkbox/src/checkbox.styles.ts

```diff
@@ -1,5 +1,18 @@
+import { featureVersion } from '@spryker-oryx/utilities';
+
 import { css } from 'lit';
 
+const focusStyles =
+  featureVersion >= '1.4'
+    ? css`
+        ::slotted(input:focus-visible)::after {
+          outline: solid 1px var(--oryx-color-focus);
+          outline-offset: 4px;
+          border-radius: 1px;
+        }
+      `
+    : (`` as any);
```

Comment:
```suggestion
    :  css``;
```

Reply:
won't work as it throws `Unexpected empty source  no-empty-source` in stylelint. This only works if it holds style rules. I've tried many solutions and ended up with this one.